src/Entity/Adherent.php

```diff
@@ -148,11 +149,11 @@ class Adherent implements UserInterface, GeoPointInterface, EncoderAwareInterfac
     private $managedArea;
 
     /**
-     * @var CoordinatorManagedArea|null
+     * @var CoordinatorManagedArea[]|Collection|null
      *
-     * @ORM\OneToOne(targetEntity="AppBundle\Entity\CoordinatorManagedArea", mappedBy="adherent", cascade={"persist"})
+     * @ORM\OneToMany(targetEntity="AppBundle\Entity\CoordinatorManagedArea", mappedBy="adherent", cascade={"persist", "remove"}, orphanRemoval=true)
```

Comment:
Since a user can't be a coordinator of both `CitizenProject` and `Comittee` sectors, why is this change needed ?

Reply:
@Remg, he must be able in code to be a coordinator of both sectors: cp and committee, but today we must allow only one choice (cp OR committee sector) in admin panel. In the future, we have just to activate `allow_add` & `allow_delete` options on admin form to allow to set many sectors and that's all.